spec.md

```diff
@@ -99,7 +97,8 @@ The `Span` interface must have the following capabilities:
 
 The `Tracer` interface must have the following capabilities:
 
-- Start a `Span` that has no parent, commonly referred to as a *root* `Span` **(py: `start_trace`, go: `StartTrace`)**
+- Start a new child `Span` with a given operation name. With no parent, commonly referred to as a *root* `Span`, no trace attributes exist. With a parent any trace attributes must be passed through into the created child. **(py: `create_span`, go: `CreateSpan`)**
```

Comment:
> Therefore I think this is worth supporting by having two span creation methods: one which signals to the tracer that a unit of sample-able work is beginning, and one that does not.

I definitely wouldn't advocate for such a burden upon the end-user.

I see your point in that `tracer.create_span(optional<span>)` fails to distinguish between a null argument that triggers a sampling logic decision and a null argument that prevents sampling.

> how would I implement a case where I would like to start a new Span only if a previously existing trace context (eg. upstream span) exists?

It was always my understanding that this is the configured sampling logic supplied to the tracer within the current process.

For example something along the lines of 
   
    def process_initialize
        sampling = Sampling.NO_NEW_TRACES
        tracer = Tracer.create(sampling)

I wouldn't think that where code is instrumented needs to be pollute with such decision making.